File: client-sdk/csharp/sdk-client/src/VertexCache/SdkClient/ConsoleApp/ConsoleDisplay.cs
using VertexCache.Sdk.Core;

namespace VertexCache.SdkClient.ConsoleApp
{
    public static class ConsoleDisplay
    {
        public static void PrintWelcomeBanner(VertexCacheSdkOptions options)
        {
            System.Console.Clear();
            System.Console.WriteLine("VertexCache C# Client Console:");
            System.Console.WriteLine($"  Host: {options.ServerHost}");
            System.Console.WriteLine($"  Port: {options.ServerPort}");
            System.Console.WriteLine($"  Message Layer Encryption Enabled: {(options.EnableEncryption ? "Yes" : "No")}");
            System.Console.WriteLine($"  Transport Layer Encryption Enabled: {(options.EnableEncryptionTransport ? "Yes" : "No")}");
            System.Console.WriteLine($"  Transport Layer Verify Certificate: {(options.EnableVerifyCertificate ? "Yes" : "No")}");
            System.Console.WriteLine();
        }

        public static void PrintPrompt(VertexCacheSdkOptions options)
        {
            System.Console.Write($"VertexCache Console, {options.ServerHost}:{options.ServerPort}> ");
        }
    }
}

File: client-sdk/go/sdk-client/console_app/console_display.go
package console_app

import (
	"fmt"
	"sdk/core"
)

func PrintWelcomeBanner(options *core.VertexCacheSdkOptions) {
	clearConsole()
	fmt.Println("VertexCache Go Client Console:")
	fmt.Printf("  Host: %s\n", options.ServerHost)
	fmt.Printf("  Port: %d\n", options.ServerPort)
	fmt.Printf("  Message Layer Encryption Enabled: %s\n", boolToYesNo(options.EnableEncryption))
	fmt.Printf("  Transport Layer Encryption Enabled: %s\n", boolToYesNo(options.EnableEncryptionTransport))
	fmt.Printf("  Transport Layer Verify Certificate: %s\n", boolToYesNo(options.EnableVerifyCertificate))
	fmt.Println()
}

func PrintPrompt(options *core.VertexCacheSdkOptions) {
	fmt.Printf("VertexCache Console, %s:%d> ", options.ServerHost, options.ServerPort)
}

func boolToYesNo(value bool) string {
	if value {
		return "Yes"
	}
	return "No"
}

func clearConsole() {
	// Only works reliably on Unix/macOS terminals
	fmt.Print("\033[H\033[2J")
}
